suite-native/transactions/src/components/TransactionDetail/TransactionDetailParametersSheet.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import * as Clipboard from 'expo-clipboard';
+
+import { Blockchain, WalletAccountTransaction } from '@suite-common/wallet-types/src';
+import { Box, Card, IconButton, Text, VStack } from '@suite-native/atoms/src';
+import { Icon } from '@trezor/icons/src';
+import { getConfirmations, getFeeRate, getFeeUnits } from '@suite-common/wallet-utils';
+
+import { TransactionDetailSheet } from './TransactionDetailSheet';
+import { TransactionDetailRow } from './TransactionDetailRow';
+
+type TransactionDetailParametersSheetProps = {
+    isVisible: boolean;
+    transaction: WalletAccountTransaction;
+    handleSheetVisibility: () => void;
+    blockchain: Blockchain;
+};
+export const TransactionDetailParametersSheet = ({
+    isVisible,
+    handleSheetVisibility,
+    transaction,
+    blockchain,
+}: TransactionDetailParametersSheetProps) => {
+    const handleClickCopy = () => Clipboard.setStringAsync(transaction.txid);
+    const formattedTransactionId = `#${transaction.txid.slice(0, 11)}...`;
+    const confirmationsCnt = getConfirmations(transaction, blockchain.blockHeight);
```

Comment:
Don't use shortcuts like this just to save 2 letters and make it less readable pls 😅

```suggestion
    const confirmationsCount = getConfirmations(transaction, blockchain.blockHeight);
```

Reply:
fixed